validator/client/runner.go

```diff
@@ -67,9 +73,13 @@ func run(ctx context.Context, v Validator) {
 	if err := v.WaitForSync(ctx); err != nil {
 		log.Fatalf("Could not determine if beacon node synced: %v", err)
 	}
-	if err := v.WaitForActivation(ctx); err != nil {
+
+	accountsChangedChan := make(chan struct{})
```

Comment:
Any reason this is an unbuffered channel? It will block on write if there is no listener ready (although seems like it's ok in this case)